src/payment/PaymentPage.jsx

```diff
@@ -71,11 +74,16 @@ class PaymentPage extends React.Component {
         {loading ? this.renderLoading() : null}
         {loaded ? (
           <div className="row">
-            <div className="col-md-5 pr-md-5 col-basket-summary">
-              {isEmpty ? this.renderBasket() : this.renderEmptyMessage()}
-            </div>
+            { isEmpty ?
+              <div className="col-md-5 pr-md-5 col-basket-summary">
+                {this.renderBasket()}
+              </div> :
+              <div>
+                {this.renderEmptyMessage()}
+              </div>
+            }
             <div className="col-md-7 pl-md-5">
-              <PaymentForm />
+              {isEmpty ? <PaymentForm /> : null}
```

Comment:
Instead of rendering the empty message just in the left hand column and then hiding the payment form, let's move this `isEmpty` check so we can render either:
```
 <Row>
    <Col>
      <Basket />
    </Col>
    <Col>
       <Form />
    </Col>
</Row>
```
Or
```
<div>
   <Empty>
</div>
```